internal/plugins/golang/init.go

```diff
@@ -43,11 +50,118 @@ func (p *initPlugin) Run() error {
 		return err
 	}
 
+	// Update the scaffolded Makefile with operator-sdk recipes.
+	// TODO: rewrite this when plugins phase 2 is implemented.
+	if err := initUpdateMakefile(); err != nil {
+		return fmt.Errorf("error updating Makefile: %v", err)
+	}
+
 	// Update plugin config section with this plugin's configuration.
 	cfg := Config{}
 	if err := p.config.EncodePluginConfig(pluginConfigKey, cfg); err != nil {
 		return fmt.Errorf("error writing plugin config for %s: %v", pluginConfigKey, err)
 	}
 
+	// Write a kustomization.yaml to the config directory.
+	if err := kustomize.Write(filepath.Join("config", "bundle"), bundleKustomization); err != nil {
+		return err
+	}
+
```

Comment:
Could we not centralize the code in a bundle.go as we have the config.go and here just call 1 method?